src/app/helpers/tag-manager.js

```diff
@@ -22,23 +22,7 @@ const tagManagerID = 'GTM-W6N7PB';
     }
 })(window, document, 'script', 'dataLayer', tagManagerID);
 
-// GA4 stuff
-const {GA4: ga4id} = window.SETTINGS;
-
-if (ga4id) {
-    const tag = document.createElement('script');
-
-    tag.type = 'text/javascript';
-    tag.onload = () => {
-        window.dataLayer = window.dataLayer || [];
-
-        function gtag(...args) {
-            window.dataLayer.push(args);
-        }
-
-        gtag('js', new Date());
-        gtag('config', ga4id);
-    };
-    tag.src = `https://www.googletagmanager.com/gtag/js?id=${ga4id}`;
-    document.body.appendChild(tag);
-}
+window.dataLayer = window.dataLayer || [];
+// eslint-disable-next-line prefer-rest-params
+window.gtag = window.gtag || function () {window.dataLayer.push(arguments);};
```

Comment:
Why not use the rest params (`...args`)?

Reply:
there is some magic going on here, because calling `dataLayer.push` directly doesn't behave the same way as calling gtag. so i figured we should do it as close the [the actual instructions](https://developers.google.com/tag-platform/gtagjs/install) as possible. i'm actually more scared about the function not having a name, but it seems to work this way and i was having some problem i forget with doing it the other way